notifications/src/main/java/org/openedx/notifications/presentation/primer/NotificationsPrimerDialogFragment.kt

```diff
@@ -82,15 +84,11 @@ class NotificationsPrimerDialogFragment : DialogFragment() {
                                 viewModel.dismissDialog()
                             },
                             onNotifyClick = {
-                                viewModel.hideDialog()
```

Comment:
Any reason to remove it? We need to hide the Primer in case the System Permission Dialog is visible.

Reply:
reverted.